bin/identity-tp

```diff
@@ -13,24 +13,28 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# ------------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 
-import os
-import sys
-import sysconfig
-
-build_str = "lib.{}-{}.{}".format(
-    sysconfig.get_platform(),
-    sys.version_info.major, sys.version_info.minor)
-
-sys.path.insert(0, os.path.join(
-    os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
-    'sdk', 'python'))
-sys.path.insert(0, os.path.join(
-    os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
-    'families', 'identity', 'python'))
-
-from sawtooth_identity.processor.main import main
+# Copyright 2018 Bitwise IO, Inc.
```

Comment:
This line should go below `Copyright 2017 Intel Corporation` and the rest of this license should be removed